package-lock.json

```diff
@@ -430,13 +430,13 @@
     },
     "@humanwhocodes/gitignore-to-minimatch": {
       "version": "1.0.2",
-      "resolved": "https://trynow-352765517868.d.codeartifact.us-east-1.amazonaws.com:443/npm/trynow-nodejs/@humanwhocodes/gitignore-to-minimatch/-/gitignore-to-minimatch-1.0.2.tgz",
+      "resolved": "https://registry.npmjs.org/@humanwhocodes/gitignore-to-minimatch/-/gitignore-to-minimatch-1.0.2.tgz",
       "integrity": "sha512-rSqmMJDdLFUsyxR6FMtD00nfQKKLFb1kv+qBbOVKqErvloEIJLo5bDTJTQNTYgeyp78JsA7u/NPi5jT1GR/MuA==",
       "dev": true
     },
     "@humanwhocodes/object-schema": {
       "version": "1.2.1",
-      "resolved": "https://trynow-352765517868.d.codeartifact.us-east-1.amazonaws.com:443/npm/trynow-nodejs/@humanwhocodes/object-schema/-/object-schema-1.2.1.tgz",
+      "resolved": "https://registry.npmjs.org/@humanwhocodes/object-schema/-/object-schema-1.2.1.tgz",
```

Comment:
@joshrathke just a heads up, when 0.0.11 was cut, a lot of these got changed to internal CA URLs.